apps/web/src/actions/copilot/requestSuggestion.ts

```diff
@@ -62,6 +62,7 @@ export const requestSuggestionAction = authProcedure
     }).then((r) => r.unwrap())
 
     const result = await sdk.run(env.COPILOT_CODE_SUGGESTION_PROMPT_PATH, {
+      stream: true,
```

Comment:
I moved all our internal use of SDK to work with new code. So all cases were using streaming and should keep using streaming. 

Please let me know if I forgot any

Reply:
Ok, setting to false